frontend/src/components/landing/Landing.tsx

```diff
@@ -0,0 +1,141 @@
+import React, { useState, ReactElement } from "react";
+import cx from "classnames";
+import styles from "./Landing.module.css";
+
+import Button from "../atoms/button/Button";
+
+import LandingPageTemplate from "../../pages/landingPageTemplate/LandingPageTemplate";
+import introScreen from "../../assets/images/intro-screen.png";
+import registrationScreen from "../../assets/images/registration-screen.png";
+import upgradePage from "../../assets/images/upgrade-page.png";
+import keyboardScreen from "../../assets/images/keyboard-screen.png";
+
+import getFurtherUrl from "../../services/getFurtherUrl";
+import getPythonSDKDocsUrl from "../../services/getPythonSDKDocsUrl";
+import openKnowledgeBase from "../../services/openKnowledgeBase";
+import openFurther from "../../services/openFurther";
+import openPythonSDKDocs from "../../services/openPythonSDKDocs";
+
+export type UrlData = {
+  defaultUrl: string;
+  urlService?: () => Promise<{ [s: string]: string }>;
+  onWebRenderer: () => Promise<void>;
+};
+
+export type LandingPageElement = {
+  title: string;
+  message: string;
+  prompt: ReactElement;
+  image: string;
+  urlInfo: UrlData;
+  buttonLabel?: string;
+};
```

Comment:
This can be removed since it is only used in one place now as the LandingPageTemplate component's props.